api/MLGraphBuilder.json

```diff
@@ -105,41 +105,21 @@
             "web-features:webnn"
           ],
           "support": {
-            "chrome": [
-              {
-                "version_added": "126",
-                "flags": [
-                  {
-                    "name": "#web-machine-learning-neural-network",
-                    "type": "preference",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "notes": "Supported on CPUs, GPUs and NPUs on macOS."
-              },
-              {
-                "version_added": "121",
-                "flags": [
-                  {
-                    "name": "#web-machine-learning-neural-network",
-                    "type": "preference",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "notes": "Supported on GPUs and NPUs on Windows."
-              },
-              {
-                "version_added": "116",
-                "flags": [
-                  {
-                    "name": "#web-machine-learning-neural-network",
-                    "type": "preference",
-                    "value_to_set": "Enabled"
-                  }
-                ],
-                "notes": "Supported on CPUs on Windows, ChromeOS and Linux."
-              }
-            ],
+            "chrome": {
+              "version_added": "126",
+              "flags": [
+                {
+                  "name": "#web-machine-learning-neural-network",
+                  "type": "preference",
+                  "value_to_set": "Enabled"
+                }
+              ],
+              "notes": [
+                "Supported on CPUs, GPUs and NPUs on macOS.",
+                "Supported on GPUs and NPUs on Windows.",
+                "Supported on CPUs on Windows, ChromeOS and Linux."
+              ]
+            },
```

Comment:
There are statements that were accidentally removed.

Reply:
Found the culprit:

https://github.com/mdn/browser-compat-data/blob/39d2eb373516e790ef560323480b089be7d49d1c/lint/linter/test-flags.ts#L54-L57

So we mustn't automatically set `version_removed` on a statement with `flags`.upda

Reply:
Fixed via dde75ca89d6a19f08a1cc33ebab6c3fad418250e.